src/data/map.js

```diff
@@ -226,7 +226,7 @@ const getPokemon = async (minLat, maxLat, minLon, maxLon, showIV, updated, pokem
             let displayPokemonId;
             let pvpRankingsGreatLeague;
             let pvpRankingsUltraLeague;
-            if (showIV) {
+            if (showPVP) {
                 atkIv = result.atk_iv;
                 defIv = result.def_iv;
                 staIv = result.sta_iv;
```

Comment:
```suggestion
```
Github won't let me quote/suggest code changes to the below 6 lines, also remove them
```diff
- 233                move1 = result.move_1;
- 234                move2 = result.move_2;
- 235                cp = result.cp;
- 236                level = result.level;
- 237                weight = result.weight;
- 238                size = result.size;
```
The following should just be left after the suggestions:
```js
            if (showIV) {
                atkIv = result.atk_iv;
                defIv = result.def_iv;
                staIv = result.sta_iv;
                move1 = result.move_1;
                move2 = result.move_2;
                cp = result.cp;
                level = result.level;
                weight = result.weight;
                size = result.size;
                displayPokemonId = result.display_pokemon_id;
            } else {
                atkIv = null;
                defIv = null;
                staIv = null;
                move1 = null;
                move2 = null;
                cp = null;
                level = null;
                weight = null;
                size = null;
                displayPokemonId = null;
            }
            if (showPVP) {
                pvpRankingsGreatLeague = JSON.parse(result.pvp_rankings_great_league);
                pvpRankingsUltraLeague = JSON.parse(result.pvp_rankings_ultra_league);
            } else {
                pvpRankingsGreatLeague = null;
                pvpRankingsUltraLeague = null;
            }
```